adapters_l1_attributes_test.go

```diff
@@ -0,0 +1,159 @@
+package monomer_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/polymerdao/monomer"
+	"github.com/polymerdao/monomer/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsL1AttributesTx(t *testing.T) {
+	l1InfoTx, depositTx, cosmosEthTx := testutils.GenerateEthTxs(t)
+	l1InfoTx.GasPrice().Set(new(big.Int).SetUint64(derive.RegolithSystemTxGas))
+
+	// https://optimistic.etherscan.io/tx/0xc104db2f3b40528b113469478a26d241567f7b3f8b821d443a4fd97307db908d
+	l1AttributesTxData := common.FromHex("0x440a5e200000146b000f79c500000000000000020000000066c89abf00000000013a34f100000000000000000000000000000000000000000000000000000001fe59ec8400000000000000000000000000000000000000000000000000000000000000016f1081e14e0b922d22358bd92f3d994a93e8b0760a8dcf5a5eaa84e576bf18300000000000000000000000006887246668a3b87f54deb3b94ba47a6f63f32985")
```

Comment:
If we aren't processing the data (and I think we are not ?), then it might be more transparent to readers here if we used obvious dummy data (0x123abc).